pkg/mgrconfig/load.go

```diff
@@ -364,7 +364,8 @@ func ParseEnabledSyscalls(target *prog.Target, enabled, disabled []string,
 	for call := range syscalls {
 		if target.Syscalls[call].Attrs.Disabled ||
 			descriptionsMode == ManualDescriptions && target.Syscalls[call].Attrs.Automatic ||
-			descriptionsMode == AutoDescriptions && !target.Syscalls[call].Attrs.Automatic {
+			descriptionsMode == AutoDescriptions &&
+				!target.Syscalls[call].Attrs.Automatic && !target.Syscalls[call].Attrs.AutomaticHelper {
```

Comment:
Nit: do you think it would make sense to add a check that no call has both `automatic` and `automatic_helper` set at the same time?

FWIW here's where the check could be https://github.com/google/syzkaller/blob/073f8be214232113bbf75d16a9bb0372beee85b9/pkg/compiler/check.go#L372